### ISSUE TITLE
Add defaults to the description with -h? 

### ISSUE THREAD

**Issue by WORMSS** [nodeca/argparse#97]
So when you do -h it can show what the current defaults are?

``` batch
Optional arguments:
  -h, --help     Show this help message and exit.
  -v, --version  Show program's version number and exit.
  --port PORT    Port Number (8080)
  --type TYPE    Location of Type File (scripts/type.js)
```

Doesn't have to be in brackets, it's just how I have currently added it.
For now I have added them directly into the "help" of the addArgument, but I thought if it could display the defaults by itself it would be nice. That way I don't have to change the default value in 2 locations.

**Comment by piranna** [comment]
It makes sense, I like it.

**Comment by puzrin** [comment]
@piranna we support only critical bugfixes now. New maintaier needed to refresh port and so on.

**Comment by piranna** [comment]
Would be a pull-request accepted?

**Comment by puzrin** [comment]
@piranna that's discussable. Personally, i'd prefer "atomic" updates to certain version of python's parser. Because that simplifies feature tracking and maintenance.

But if something is very important, it can be accepted separate. Features not from mainstream are declined.

**Comment by piranna** [comment]
Python ArgParse support this with the [ArgumentDefaultsHelpFormatter](https://docs.python.org/3/library/argparse.html#argparse.ArgumentDefaultsHelpFormatter) `formatter_class` option.

**Comment by ritiek** [comment]
Any news on this?

**Comment by rlidwka** [comment]
We've ported everything from scratch based on python 3.9 (to be released as argparse version 2.0).

So this should now be supported (exactly the same way as in python).
